data/styles/clear/include/Basemap.mapcss

```diff
@@ -445,6 +446,7 @@ area|z14-[area:highway=living_street],
 area|z14-[amenity=grave_yard],
 area|z14-[landuse=cemetery],
 area|z14-[amenity=university],
+area|z14-[amenity=school],
```

Comment:
School grounds are usually much smaller compared to universities, so makes sense to render them at higher zoom levels.
E.g. in Icons.mapcss schools icons are rendered starting from z17-. I'd keep the area fill and icons either in sync or add the fill one zoom level earlier maybe.